src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/AddressPhone.kt

```diff
@@ -0,0 +1,31 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.helper.builders
+
+import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.AddressPhone
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.PersonAddress
+
+@DslMarker
+annotation class AddressPhoneDslMarker
+
+@NomisDataDslMarker
+interface AddressPhoneDsl
+
+@Component
+class AddressPhoneBuilderFactory {
+  fun builder() = AddressPhoneBuilderRepositoryBuilder()
+}
+
+class AddressPhoneBuilderRepositoryBuilder : AddressPhoneDsl {
```

Comment:
Can phone numbers also be attached directly to a person (rather than an address)?